src/tags.h

```diff
@@ -210,13 +210,22 @@ struct DataDef {
     using type = Dataset;
     static constexpr auto unit = Unit::Id::Dimensionless;
   };
+  struct EventTofs {
+    using type = boost::container::small_vector<double, 8>;
+    static constexpr auto unit = Unit::Id::Dimensionless;
+  };
+  struct EventPulseTimes {
+    using type = boost::container::small_vector<double, 8>;
```

Comment:
Is double the right type for this? I imagine we would want to store as unsigned ints (64 bit). Nanoseconds from epoch. Same would apply to TOF.

Reply:
It can be whatever you like since the type can be customized. Yet another option would be to simply store a (smaller) pulse index and provide a separate table of pulse times, shared by all event lists.